licenses.d2iq.yaml

```diff
@@ -355,6 +355,11 @@ resources:
       - license_path: LICENSE
         ref: release-1.18
         url: https://github.com/knative/pkg
+  - container_image: gcr.io/knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate:v1.18.1
```

Comment:
```suggestion

```

Reply:
The one that's failing is this migrate one:
https://jira.nutanix.com/browse/NCN-109043

```sh
  Warning  Failed     61s                    kubelet            Failed to pull image "gcr.io/knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate:v1.18.1": [failed to pull and unpack image "gcr.io/knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate:v1.18.1": failed to resolve reference "gcr.io/knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate:v1.18.1": failed to do request...
```